keyboards/handwired/dactyl_manuform/5x8/keymaps/default/keymap.c

```diff
@@ -0,0 +1,38 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _FN     1
+#define _NUMPAD 2
+
+// Some basic macros
+#define TASK   LCTL(LSFT(KC_ESC))
+#define TAB_R  LCTL(KC_TAB)
+#define TAB_L  LCTL(LSFT(KC_TAB))
+#define TAB_RO LCTL(LSFT(KC_T))
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_QWERTY] = LAYOUT_5x8(
```

Comment:
This layout should be rewritten such that the halves are not stacked. See the rest of the layouts in `handwired/dactyl_manuform/`.

Reply:
i rewrote this code

Reply:
> This layout should be rewritten such that the halves are not stacked. See the rest of the layouts in handwired/dactyl_manuform/.

Reply:
I rewrote the layout so that the halves don't overlap.